src/NLog.Extensions.Logging/NLog.Extensions.Logging.csproj

```diff
@@ -5,26 +5,28 @@
     <TargetFrameworks>net451;net461;netstandard1.3;netstandard1.5;netstandard2.0</TargetFrameworks>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    
+
     <Product>NLog.Extensions.Logging v$(ProductVersion)</Product>
     <InformationalVersion>$(ProductVersion)</InformationalVersion>
     <Authors>Microsoft;Julian Verdurmen</Authors>
     <Company>NLog</Company>
-    <Description>NLog provider for Microsoft.Extensions.Logging for usage in .NET Standard libraries and console applicaties.
+    <Description>
+      NLog provider for Microsoft.Extensions.Logging for usage in .NET Standard libraries and console applicaties.
 
-For ASP.NET Core, use NLog.Web.AspNetCore: https://www.nuget.org/packages/NLog.Web.AspNetCore
+      For ASP.NET Core, use NLog.Web.AspNetCore: https://www.nuget.org/packages/NLog.Web.AspNetCore
     </Description>
     <PackageTags>NLog;Microsoft.Extensions.Logging;log;logfiles;netcore;nlog</PackageTags>
     <PackageReleaseNotes>
 
-        1.2: BeginScope accepts any type of KeyValuePair and Dictionary  (@snakefoot)
+      1.1: Added support for using NLog MessageTemplate Parser (NLogProviderOptions.ParseMessageTemplates)  (@snakefoot)
```

Comment:
I think this is a git mistake? It reverts the to an older version

Reply:
I think the git diff is confused, looks ok in the full source. Or maybe i'm lysdexic.